libcst/_visitors.py

```diff
@@ -71,8 +72,32 @@ def on_leave(
 
         return updated_node
 
+    def on_visit_attribute(self, node: "CSTNode", attribute: str) -> None:
```

Comment:
The `on_visit_attribute` and `on_leave_attribute` code is shared between the visitor and transformer. Do we want to factor it out into helper functions to reuse code?

Reply:
Alternatively could they be moved into `CSTTypedVisitorFunctions`?